templates/panels-development.ini.erb

```diff
@@ -0,0 +1,23 @@
+# THIS FILE IS GENERATED BY PUPPET
+# DO NOT EDIT DIRECTLY, INSTEAD EDIT THE .erb template
+
+<% if @hide_schedule %>
+hide_schedule = <%= @hide_schedule %>
+<% end -%>
+
+<% if @expected_response %>
+expected_response = "<%= @expected_response %>"
+<% end -%>
+
+<% if @event_location %>
+event_location = "<%= @event_location %>"
+<% end -%>
+
+<% if @panel_rooms %>
```

Comment:
Yea, it ends up generating the following line in the INI file because our empty list is truthy in ruby:

puppet config for magstock says:
```
uber::plugin_panels::panel_rooms: ","
```

INI looks like:
```
panel_rooms = ,
```

If puppet doesn't specify anything, puppet's deftault is 'undef' which means that the configspec.ini value for panel_rooms of ```"panels_1", "panels_2", "panels_3", "panels_4"``` will be used instead.